watertap/unit_models/electrodialysis_0D.py

```diff
@@ -319,6 +319,14 @@ def build(self):
             units=pyunits.kW * pyunits.hour * pyunits.meter**-3,
             doc="Diluate-volume-flow-rate-specific electrical power consumption",
         )
+        self.water_recovery_mass = Var(
```

Comment:
`recovery_mass_H2O` might be more consistent with the way we named water recovery elsewhere in WaterTAP. We also have `recovery_mass_phase_comp`, indexed by phase and component, as well as `recovery_vol_phase`.

Reply:
I changed the name for mass.   I will add the other two when needed in the future.